content/guides/mobile-principles/principles-optimize-minify-compression.md

```diff
@@ -42,12 +42,12 @@ mod_gzip_item_include mime ^text/.*
 mod_gzip_item_include mime ^application/x-javascript.*
 mod_gzip_item_exclude mime ^image/.*
 mod_gzip_item_exclude rspheader ^Content-Encoding:.*gzip.*
-&#60; /ifModule&#62;
-</pre>
+</ifModule>
+```
 
 **Apache Server** — Enable the zip parameters in the .htaccess file. Below is an example:
```

Comment:
We should also prefer `.htaccess` to be consistent.

Reply:
updated with `` around .htaccess